libs/sdk-ui-dashboard/src/presentation/widget/common/configuration/ConfigurationBubble.tsx

```diff
@@ -55,10 +56,7 @@ export const ConfigurationBubble: React.FC<IConfigurationBubbleProps> = (props)
             arrowDirections={arrowDirections}
         >
             {children}
-            {/* <ConfigurationPanel
-                onWidgetDelete={props.onWidgetDelete}
-                // handleSetConfigMenuDisplay={setConfigMenuDisplay}
-            /> */}
+            <ConfigurationPanel />
```

Comment:
Will this break KPI configuration? The KD was a mangled mess where KPI and visualisation config are in one component, we should keep them separated if possible for once we remove KPI altogether in the future.